Sources/Responses/SignRequestSigner.swift

```diff
@@ -77,6 +77,22 @@ public class SignRequestSigner: BoxModel {
     public let redirectUrl: String?
     /// The URL that a signer will be redirect to after declining to sign a document.
     public let declinedRedirectUrl: String?
+    /// This URL is specifically designed for  signing documents within an HTML `iframe` tag.
```

Comment:
Extra space between "for" and "signing". I saw it's like this in the spec so maybe we shouldn't care about it.